src/main/proto/Report.proto

```diff
@@ -42,7 +42,12 @@ message AssignReport {
     string stride_API_Token = 4;
     string stride_cloud_id = 5;
     string stride_conversation_id = 6;
-    repeated Email email_list = 7;
+    CommunicationList communication_list = 7;
+}
+
+message CommunicationList{
+	repeated Email email = 1;
+	repeated int32 teams = 2;
```

Comment:
what is the possible value for this field?

Reply:
> what is the possible value for this field?

for Email possible is user_email and user_name and for teams its integer array

Reply:
could you bring an example of a `teams` value? What do these integers represent?